app/src/main/java/be/chvp/nanoledger/ui/common/TransactionFormViewModel.kt

```diff
@@ -174,6 +174,10 @@ abstract class TransactionFormViewModel
                     transaction.append(
                         "    ${account}${note}\n",
                     )
+                } else if (currency == "") {
+                    transaction.append(
+                        "    $account  $spaces$quantity$note\n",
```

Comment:
To make sure the alignment is still correct:
```suggestion
                        "    $account  $spaces $quantity$note\n",
```